infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -6,16 +6,14 @@ server {
 server {
     listen 443;
     server_name {{ inventory_hostname }};
-
-    ssl_certificate           {{ nginx_ssl_certificate }};
-    ssl_certificate_key       {{ nginx_ssl_certificate_key }};
+    ssl_certificate /etc/letsencrypt/live/{{ inventory_hostname }}/fullchain.pem; # managed by Certbot
+    ssl_certificate_key /etc/letsencrypt/live/{{ inventory_hostname }}/privkey.pem; # managed by Certbot
 
     ssl on;
     ssl_session_cache  builtin:1000  shared:SSL:10m;
-    ssl_protocols  TLSv1.1 TLSv1.2;
-    ssl_ciphers HIGH:!aNULL:!eNULL:!EXPORT:!CAMELLIA:!DES:!MD5:!PSK:!RC4;
+    ssl_protocols  TLSv1.2 TLSv1.3;
+    ssl_ciphers EECDH+AESGCM:EDH+AESGCM;
```

Comment:
Excellent change, but it would be nice to see the other changes as well suggested on https://cipherli.st/ since we have the benefit of being able to control both the client and the server 👍 


I'm not sure if our current server supports TLS 1.3, depends on the Ubuntu version I guess.
In any case I'm pretty sure it doesn't work on the forum server.

Reply:
Should be easy to do, any objection if I add those in as a follow-up?

Reply:
No objection, adding in the dhparams is requires an extra key generation step anyways, or making them static and putting it in the repository shouldn't be a problem either (see https://security.stackexchange.com/questions/94390/whats-the-purpose-of-dh-parameters )